client/src/components/subscription/subscriptionsList.js

```diff
@@ -97,17 +102,25 @@ class SubscriptionsList extends React.Component {
           }
           {editingSubscription === subscriptionUuid && 
             <div className="card p-3 m-2 d-flex flex-wrap">
-              <UpdateSubscription updateSubscription={this.handleUpdate} toggleLoadingState={this.toggleLoadingState} currentSubscriptions={subscriptions} prevSubscription={subscription} />
+              <UpdateSubscription
+                updateSubscription={this.handleUpdate}
+                showSubscriptionList={this.showSubscriptionList}
+                toggleLoadingState={this.toggleLoadingState}
+                currentSubscriptions={subscriptions}
+                prevSubscription={subscription} />
               <button onClick={() => this.setState({ editingSubscription: null })} className="btn btn-link my-2" type="button">Cancel</button>
             </div>
             }
         </div>
       );
     });
 
-    const addSubScriptionTemplates = addingSubscription
+    const addSubscriptionTemplates = addingSubscription
       ? <div className="card p-3 m-2 d-flex flex-column">
-          <CreateSubscription addSubscription={this.handleUpdate} toggleLoadingState={this.toggleLoadingState} currentSubscriptions={subscriptions} />
+          <CreateSubscription addSubscription={this.handleUpdate}
```

Comment:
Should we keep syntax the same as UpdateSubscription, so put addSubscription on newline?
```suggestion
          <CreateSubscription 
            addSubscription={this.handleUpdate}
```

Reply:
oh yeahh that's fine. haha this is where having prettier for auto-formatting comes in handy